package.json

```
@@ -81,8 +81,8 @@
   "devDependencies": {
     "@inrupt/eslint-config-base": "^1.0.0",
     "@inrupt/eslint-config-lib": "^1.0.0",
```

```suggestion
    "@inrupt/eslint-config-lib": "^1.0.0",
```

You no longer need `base` here.

---

Actually, ignore that. This repo hasn't been correctly upgraded to 1.0.0